README.md

```diff
@@ -1,2 +1,15 @@
 # Portfolio-Webstie
 This portfolio website is built to demonstrate my skills by showing the project built
+
+Built With Languages
+Html
+CSS
+
+Author 
+Stephen Annor 
+
+Accounts 
+Github: https://github.com/braincee
```

Comment:
Kindly add the missing headers in your readme file. Use this [template for reference](https://github.com/microverseinc/readme-template/blob/master/README.md)

[**optional**] For headers that might be redundant, kindly leave them out, for example, **Install and Run Test**.